pkg/vmimage/data/etc/kdump.conf

```diff
@@ -0,0 +1,2 @@
+path /var/crash
```

Comment:
isn't the default /etc/kdump.conf good enough?

Reply:
those are the default values, I just removed comments

Reply:
don't bother overwriting the default configuration if it's fine